infrastructure/docker/services/php/Dockerfile

```diff
@@ -69,10 +69,6 @@ CMD ["runsvdir", "-P", "/etc/service"]
 
 FROM php-base as worker
 
```

Comment:
```suggestion

```
is this line still needed?

Reply:
It's referenced here I guess https://github.com/jolicode/docker-starter/pull/219/files#diff-ff35e32a193dad01774fc1e637b7896b2ad469a51cf9fdc4516f5ce3ac250536L8